plugins/by-name/cord-nvim/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  helpers,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin {
+  name = "cord-nvim";
+  originalName = "cord.nvim";
+  package = "cord-nvim";
+  maintainers = [ lib.maintainers.eveeifyeve ];
+
+  settingsOptions = {
+    usercmds = helpers.defaultNullOpts.mkBool false ''
+      Enables user commands
+    '';
+    log_level =
+      helpers.defaultNullOpts.mkEnum
+        [
+          "trace"
+          "debug"
+          "info"
+          "warn"
+          "off"
+        ]
+        null
+        ''
+          Log messages at or above this level.
+        '';
+  };
```

Comment:
```suggestion
    log_level =
        lib.nixvim.defaultNullOpts.mkLogLevel "error" ''
          Log messages at or above this level.
        '';
  };
```

Reply:
Might need to fix formatting here, hard to do from github

Reply:
I will fix the fmt when I push.